org.openhab.binding.zigbee/src/main/java/org/openhab/binding/zigbee/internal/ZigBeeNetworkStateSerializerImpl.java

```diff
@@ -180,4 +180,16 @@ public void deserialize(final ZigBeeNetworkManager networkState) {
         logger.debug("Loading ZigBee network state: Done.");
     }
 
+    public synchronized void delete() {
+        final File file = new File(networkStateFilePath + "/" + networkStateFileName + networkId + ".xml");
+        boolean networkStateExists = file.exists();
+        if (networkStateExists) {
+            if (file.delete()) {
```

Comment:
Citing from the Javadoc of `File.delete`:
> Note that the {@link java.nio.file.Files} class defines the {@link java.nio.file.Files#delete(Path) delete} method to throw an {@link IOException} when a file cannot be deleted. This is useful for error reporting and to diagnose why a file cannot be deleted.

In this light, what do you think about replacing this code with the following (will also, as a nice side effect, remove the need for the explicit `file.exists()` check:
```
try {
     Files.deleteIfExists(Paths.get(networkStateFilePath + "/" + networkStateFileName + networkId + ".xml"));
     logger.debug("Deleted network state file.");
} catch (IOException e) {
     logger.error("Error deleting network state file", e);
}
```

Reply:
Can change it as suggested.